lib/SPIRV/SPIRVReader.cpp

```diff
@@ -800,16 +800,27 @@ bool SPIRVToLLVM::postProcessOCLBuiltinWithArrayArguments(
   return true;
 }
 
-static char getTypeSuffix(Type *T) {
-  char Suffix;
+static std::string getTypeSuffix(Type *T, SPIRVType *SpirT) {
+  std::string Suffix;
 
   Type *ST = T->getScalarType();
   if (ST->isHalfTy())
-    Suffix = 'h';
+    Suffix = "h";
   else if (ST->isFloatTy())
-    Suffix = 'f';
-  else
-    Suffix = 'i';
+    Suffix = "f";
+  else {
+    SPIRVTypeInt *SIT = nullptr;
```

Comment:
What does "SIT" means? Is it "SPIRV Integer Type"?

Reply:
Yes, I'll rename it to STI, though for some reason that seemed to be a wrong thing to say :)